### ISSUE TITLE
win10下最新版迅雷会使goagent不工作

### ISSUE THREAD

**Issue by mouseroser** [phuslu/goproxy#123]
win10下最新版迅雷会使goagent不工作，花了我四天的时间，系统重装了N次了，最后才肯定是最新版的迅雷造成goagent不工作，卸载了就恢复正常了

**Comment by tylokfree** [comment]
没问题正常啊。

**Comment by raregrass** [comment]
Definitely, some softwares may cause io block in handshake of goagent ssl processing. If your CONNECT http got by goagent client didn't decode properly, it's not going to send to goagent server i guess.

**Comment by aOJzQT** [comment]
@raregrass 同意！墙国的360、QQ、百度、迅雷等这类软件有潜在风险 窃取隐私、拦截请求、把敏感信息上报给共匪特务组织
不喜欢它们，但爱看它们相互撕逼
